spec/src/main/doc/javaee/packagingdeployment_ee.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[packaging_deployment_ee]]
+
+== Packaging and deployment in Java EE
+
+
+[[bean_archive_ee]]
+=== Bean archive with EJB Session Beans
+
+When running in Java EE, the container must extend the rules defined in <<bean_archive>> with:
+
+* An implicit bean archive may also contains EJB session beans, and
+* if an archive meet the requirements to be an _implicit bean archive_ but doesn't contain any bean class with a bean defining annotation and have at least one EJB sessions beans, it's an _implicit bean archive_.
```

Comment:
"if an archive meet the requirements to be an _implicit bean archive_ but doesn't contain any bean class with a bean defining annotation" is kind of a confusing definition given that "containing classes with bean defining annotation" is itself a half of the original implicit bean archive definition.